go.mod

```diff
@@ -3,14 +3,14 @@ module github.com/okta/terraform-provider-okta
 go 1.15
 
 require (
-	github.com/bflad/tfproviderlint v0.24.0
+	github.com/bflad/tfproviderlint v0.26.0
```

Comment:
@bogdanprodan-okta  happy to revert these if you prefer!

Reply:
no, it's fine. Dependabot would create a PR on Monday anyway :)